spec/checkPullRequestReviewSpec.js

```diff
@@ -77,6 +82,58 @@ describe('Pull Request Review Module', () => {
       };
     });
 
+    describe('When reviewer requests changes and LGTM label' +
+     'was already added to the pull request.', ()=>{
+      beforeEach(async () => {
+        const label = {
+          id: 248679580,
+          node_id: 'MDU6TGFiZWwyNDg2Nzk1ODA=',
+          url: 'https://api.github.com/repos/oppia/oppia/labels/PR:%20LGTM',
+          name: 'PR: LGTM',
+          color: '009800',
+        };
+        // Set the payload action and label which will simulate removing
+        // the LGTM label.
+        payloadData.payload.action = 'labeled';
+        payloadData.payload.label = label;
+        payloadData.payload.pull_request.requested_reviewers = [
+          { login: 'reviewer1' },
+          { login: 'reviewer2' },
+        ];
```

Comment:
This doesn't look correct.

We testing that the label gets removed by oppiabot, not by the user. So we don't need to set the payload action to be labelled. 
You just need to add the lgtm label to the list of labels available in the `revewPayloadData`

Reply:
Oh Sorry I did not pay attention to that, Removed 97th line.

Reply:
You should also add the LGTM label to reviewPayloadData.labels